### PR TITLE
[MVP-1477]Create tooltip icon in react card

### PR DESCRIPTION
**Pull request by marukohao**
Create tooltip icon where users can pass in data per alert via config tool

## Test:
Untitled1
updated:
Screen Shot 2022-10-20 at 9 40 20 AM
dark mode:
Screen Shot 2022-10-21 at 6 58 20 AM

##Story:
https://notifi.atlassian.net/browse/MVP-1477

### CODE REVIEW

File: packages/notifi-react-card/lib/components/subscription/EventTypeBroadcastRow.tsx [hunk 1]
```diff
@@ -75,6 +77,10 @@ export const EventTypeBroadcastRow: React.FC<EventTypeBroadcastRowProps> = ({
     >
       <div className={clsx('EventTypeBroadcastRow__label', classNames?.label)}>
         {config.name}
+        <NotifiTooltip
```
Comment (Nathan-Notifi) [inline packages/notifi-react-card/lib/components/subscription/EventTypeBroadcastRow.tsx:80]: does this show up even if there isn't content being passed in?

Comment (marukohao) [reply packages/notifi-react-card/lib/components/subscription/EventTypeBroadcastRow.tsx]: ah, good catch, updated

File: packages/notifi-react-card/lib/components/subscription/NotifiTooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        className={clsx('Info__icon', classNames?.icon)}
+        width="14"
+        height="14"
+        viewBox="0 0 14 14"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <path
+          d="M6.33301 3.66683H7.66634V5.00016H6.33301V3.66683ZM6.33301 6.3335H7.66634V10.3335H6.33301V6.3335ZM6.99967 0.333496C3.31967 0.333496 0.333008 3.32016 0.333008 7.00016C0.333008 10.6802 3.31967 13.6668 6.99967 13.6668C10.6797 13.6668 13.6663 10.6802 13.6663 7.00016C13.6663 3.32016 10.6797 0.333496 6.99967 0.333496ZM6.99967 12.3335C4.05967 12.3335 1.66634 9.94016 1.66634 7.00016C1.66634 4.06016 4.05967 1.66683 6.99967 1.66683C9.93967 1.66683 12.333 4.06016 12.333 7.00016C12.333 9.94016 9.93967 12.3335 6.99967 12.3335Z"
+          fill="#80829D"
```
[the remark this answers is not in the document]
Comment (juni-notifi) [reply packages/notifi-react-card/lib/components/subscription/NotifiTooltip.tsx:31]: as a side-note, we can probably have `fill: var(--notifi-font-color)` or something as a default

**Review by juni-notifi — COMMENTED** [review]
Great work!!

Would you mind updating the SVG to take the fill as `inherit` and update defaults for Dark Mode? (you can test darkmode / light mode with NotifiSubscriptionCard)

Also, please update the CSS class names to reduce possibility of collisions!

File: packages/notifi-react-card/lib/components/subscription/defaults.css [hunk 3]
```diff
@@ -326,3 +330,52 @@
 .NotifiUserInfoPanel__telegramLabel {
   margin-bottom: 0 !important;
 }
+
+.Info__icon {
+  margin-left: 5px;
+  vertical-align: -webkit-baseline-middle;
```
[the remark this answers is not in the document]
Comment (marukohao) [reply packages/notifi-react-card/lib/components/subscription/defaults.css:337]: I tried to use `align-items`, but seems it doesn't work.

[the remark this answers is not in the document]
Comment (Nathan-Notifi) [reply packages/notifi-react-card/lib/components/subscription/defaults.css:337]: did you try using display: flex before using align-items as well?

[the remark this answers is not in the document]
Comment (marukohao) [reply packages/notifi-react-card/lib/components/subscription/defaults.css]: tried to using display:flex still doesn't work and it works with removing margin bottom of the icon

**Review by juni-notifi — APPROVED** [review]

**Pull request merged**